src/recv_request_vote.c

```diff
@@ -75,6 +76,12 @@ int recvRequestVote(struct raft *r,
         }
     }
 
+    /* Reject the request if we are installing a snapshot. */
+    if (replicationInstallSnapshotBusy(r)) {
+        tracef("installing snapshot -> reject");
```

Comment:
This code is not hit in a test, you can verify it by putting an `assert(false)` here.

Reply:
Yeah, that's fine, it looks safe.